biblateka/yangi_branch.md

```diff
@@ -0,0 +1,18 @@
+# Xato topib va uni boshqa branch yaratib tog'irlash
+
+Men xato topdim va uni main branchga qoshishim kerak
+buning uchun yangi branch hosil qilaman masalan pomidor nomli branch
+xatoni togirlab push qilaman
```

Comment:
Gaplar noaniq yozilgan, imlo qoidalariga rioya qilinmagan. Natijada tushunish qiyin. Iltimos to'g'irlang.